async_lru/__init__.py

```diff
@@ -159,9 +159,8 @@ def _task_done_callback(
     ) -> None:
         self.__tasks.remove(task)
 
-        if self.__ttl is not None:
+        if self.__ttl is not None and (cache_item := self.__cache.get(key)):
```

Comment:
```suggestion
        cache_item = self.__cache.get(key)
        if self.__ttl is not None and cache_item is not None:
```
I think this better matches the coding practices in our repos currently.